CODEOWNERS

```diff
@@ -46,7 +46,9 @@ k8s/aat/common/money-claims/claim-store.yaml @hmcts/cmc
 ## I&A
 k8s/**/common/ia/namespace.yaml @hmcts/immigration-asylum
 k8s/**/common/ia/identity.yaml @hmcts/immigration-asylum
-k8s/aat/common/ia/case-documents-api.yaml @hmcts/immigration-asylum
+k8s/**/common/ia/case-api.yaml @hmcts/immigration-asylum
```

Comment:
ah these should be aat until you have prod approval, only case-documents-api is approved currently

Reply:
Ok, have updated those.